guardrails/validator_base.py

```diff
@@ -174,6 +177,14 @@ class Filter:
 class Refrain:
     pass
 
+def is_word(chunk:str) -> bool:
+    return ' ' in chunk
+
+def is_sentence(chunk:str) -> bool:
+    return '.' in chunk
+
+def is_paragraph(chunk:str) -> bool:
+    return '\n' in chunk
```

Comment:
This is the right idea, I think we should instead move these to the function pattern I recommended in string_schema.

Reply:
I'm not sure what function pattern you're referring to - do you mean using @classmethod?